### PR TITLE
add gpg support to Dockerfile

### PR DESCRIPTION
**Pull request by HumairAK**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -30,5 +30,14 @@ COPY --from=ksops-builder /go/bin/kustomize /usr/local/bin/kustomize
 # Copy the plugin to kustomize plugin path
 COPY --from=ksops-builder /go/src/github.com/viaduct-ai/kustomize-sops/*  $KUSTOMIZE_PLUGIN_PATH/viaduct.ai/v1/${PKG_NAME}/
 
+RUN apt-get update && \
```
Comment (goern) [inline Dockerfile:33]: why do we want to update packages provided by the argocd community?

File: Dockerfile [hunk 2]
```diff
@@ -30,5 +30,14 @@ COPY --from=ksops-builder /go/bin/kustomize /usr/local/bin/kustomize
...
+  apt-get install -y \
+    curl \
+    gpg && \
+  apt-get clean && \
+  rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* && \
+  curl -o /usr/local/bin/sops -L https://github.com/mozilla/sops/releases/download/v3.5.0/sops-v3.5.0.linux && \
```
Comment (goern) [inline Dockerfile:39]: why do we need sops inside of the argocd image?

File: Dockerfile [hunk 3]
```diff
@@ -30,5 +30,14 @@ COPY --from=ksops-builder /go/bin/kustomize /usr/local/bin/kustomize
 # Copy the plugin to kustomize plugin path
 COPY --from=ksops-builder /go/src/github.com/viaduct-ai/kustomize-sops/*  $KUSTOMIZE_PLUGIN_PATH/viaduct.ai/v1/${PKG_NAME}/
 
+RUN apt-get update && \
+  apt-get install -y \
+    curl \
+    gpg && \
```
Comment (goern) [inline Dockerfile:36]: gpg 2.2.12 is included in the argocd image

File: Dockerfile [hunk 4]
```diff
@@ -30,5 +30,14 @@ COPY --from=ksops-builder /go/bin/kustomize /usr/local/bin/kustomize
 # Copy the plugin to kustomize plugin path
 COPY --from=ksops-builder /go/src/github.com/viaduct-ai/kustomize-sops/*  $KUSTOMIZE_PLUGIN_PATH/viaduct.ai/v1/${PKG_NAME}/
 
+RUN apt-get update && \
+  apt-get install -y \
+    curl \
```
Comment (goern) [inline Dockerfile:35]: why do we need curl?

File: Dockerfile (hunk 3 above)
Comment (HumairAK) [reply Dockerfile:36]: oh whoops I missed this - it wasn't in the original argocd last I checked, but seems to be there now

File: Dockerfile (hunk 2 above)
Comment (HumairAK) [reply Dockerfile:39]: I was thinking mostly for debugging purposes for rsh'ing into the pod and being able to manage secrets, but in hindsight, this is not really necessary

**Comment by HumairAK** [comment]
closing as gpg is already included

**Pull request closed without merging**